tools/go.mod

```diff
@@ -0,0 +1,7 @@
+module go.uber.org/goleak
```

Comment:
thanks for isolating the golang.org/x/lint dependency, much appreciated

I'm not sure this matters, but having two go.mod files claim to be the same module name is a little weird... should this be `go.uber.org/goleak/tools` or something?

Reply:
It should be, good catch! Will fix this as part of migrating off golint.